<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordGenerator</class>
 <widget class="QWidget" name="PasswordGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <widget class="QLineEdit" name="password_lineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="generatePassword_layout">
     <item>
      <layout class="QHBoxLayout" name="passwordSize_layout">
       <item>
        <widget class="QLabel" name="passwordSize_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Taille</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="passwordSize_spinbox">
         <property name="maximum">
          <number>32</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="generatePassword_button">
       <property name="text">
        <string>Relancer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="drawPassword_layout">
     <item>
      <widget class="QCheckBox" name="drawPassword_checkbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Afficher la chaine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyToClipboard_button">
       <property name="text">
        <string>Copier la chaine</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
